serde_derive/src/internals/attr.rs

```diff
@@ -341,6 +356,43 @@ impl Container {
                             }
                         }
                     }
+                } else if meta.path == RENAME_ALL_FIELDS {
+                    // #[serde(rename_all_fields = "foo")]
+                    // #[serde(rename_all_fields(serialize = "foo", deserialize = "bar"))]
+                    match item.data {
+                        syn::Data::Enum(_) => {
+                            let one_name = meta.input.peek(Token![=]);
+                            let (ser, de) = get_renames(cx, RENAME_ALL, &meta)?;
```

Comment:
```suggestion
                            let (ser, de) = get_renames(cx, RENAME_ALL_FIELDS, &meta)?;
```

Otherwise this refers to the wrong attribute name in error messages.

```console
error: malformed rename_all attribute, expected `rename_all(serialize = ..., deserialize = ...)`
 --> src/main.rs:4:27
  |
4 | #[serde(rename_all_fields(asdf = "lowercase"))]
  |                           ^^^^
```